### PR TITLE
Add Verilator dependency

### PR DESCRIPTION
**Pull request by schoeberl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,17 @@ with a clean running [Chisel3](https://www.chisel-lang.org/) project.
 
 #### JDK 8 or newer
 
-We recommend LTS releases Java 8 and Java 11. You can install the JDK as recommended by your operating system, or use the prebuilt binaries from [AdoptOpenJDK](https://adoptopenjdk.net/).
+We recommend LTS releases Java 8 and Java 11. You can install the JDK as your operating system recommends, or use the prebuilt binaries from [AdoptOpenJDK](https://adoptopenjdk.net/).
 
 #### SBT or mill
 
-SBT is the most common built tool in the Scala community. You can download it [here](https://www.scala-sbt.org/download.html).  
+SBT is the most commonly built tool in the Scala community. You can download it [here](https://www.scala-sbt.org/download.html).
```
Comment (seldridge) [inline README.md:18]:
```suggestion
SBT is the most common build tool in the Scala community. You can download it [here](https://www.scala-sbt.org/download.html).  
```

File: README.md [hunk 2]
```diff
@@ -11,13 +11,17 @@ with a clean running [Chisel3](https://www.chisel-lang.org/) project.
...
 
 #### SBT or mill
 
-SBT is the most common built tool in the Scala community. You can download it [here](https://www.scala-sbt.org/download.html).  
+SBT is the most commonly built tool in the Scala community. You can download it [here](https://www.scala-sbt.org/download.html).  
 mill is another Scala/Java build tool without obscure DSL like SBT. You can download it [here](https://github.com/com-lihaoyi/mill/releases)
 
+#### Verilator
+
+The test with `svsim` needs Verilator installed.
```
Comment (seldridge) [inline README.md:23]:
```suggestion
The test with `svsim` needs Verilator installed.
See Verilator installation instructions [here](https://verilator.org/guide/latest/install.html).
```

**Review by seldridge — APPROVED** [review]
Two comments. Mentioning Verilator explicitly makes sense to me!

**Comment by schoeberl** [comment]
Thank or adding the link.

**Comment by schoeberl** [comment]
Can you please merge? I do not have write access to this repo (or freechipsproject)

**Pull request merged**